tests/cli/commands/test_connection_command.py

```diff
@@ -321,3 +330,87 @@ def test_cli_delete_invalid_connection(self):
+                    self.assertEqual(expected_connection_uris[conn_id],
+                                     {attr: getattr(current_conn, attr)
+                                      for attr in expected_connection_uris[conn_id]})
+
+    @parameterized.expand(
+        (
+            (
+                {"CONN_ID2": [{'conn_type': 'mysql', 'host': 'host_1'},
+                              {'conn_type': 'mysql', 'host': 'host_2'}]},
+                {"CONN_ID2": [{'conn_type': 'mysql', 'host': 'host_1'},
+                              {'conn_type': 'mysql', 'host': 'host_2'}]}
+            ),
+        )
+    )
+    def test_connections_import_disposition_ignore(self, file_content, expected_connection_uris):
+        """Test connections_import command with --conflict-disposition ignore"""
+        with mock_local_file(json.dumps(file_content)):
+            connection_command.connections_import(self.parser.parse_args([
+                'connections', 'import', 'a.json']))
+
+            connection_command.connections_import(self.parser.parse_args([
+                'connections', 'import', 'a.json', '--conflict-disposition', 'ignore']))
+
+            conn_id = 'CONN_ID2'
+            with create_session() as session:
+                current_conn = session.query(Connection).filter(Connection.conn_id == conn_id).first()
+                self.assertEqual(expected_connection_uris[conn_id][0],
+                                 {attr: getattr(current_conn, attr)
+                                  for attr in expected_connection_uris[conn_id][0]})
+
+    @parameterized.expand(
+        (
+            (
+                {"CONN_ID3": [{'conn_type': 'mysql', 'host': 'host_1'},
+                              {'conn_type': 'mysql', 'host': 'host_2'}]},
+                {"CONN_ID3": [{'conn_type': 'mysql', 'host': 'host_1'},
+                              {'conn_type': 'mysql', 'host': 'host_2'}]}
+            ),
+        )
+    )
+    def test_connections_import_disposition_overwrite(self, file_content, expected_connection_uris):
+        """Test connections_import command with --conflict-disposition overwrite"""
+        with mock_local_file(json.dumps(file_content)):
+            connection_command.connections_import(self.parser.parse_args([
+                'connections', 'import', 'a.json', '--conflict-disposition', 'overwrite']))
+            connection_command.connections_import(self.parser.parse_args([
+                'connections', 'import', 'a.json', '--conflict-disposition', 'overwrite']))
+
+            with redirect_stdout(io.StringIO()) as stdout:
+                stdout = stdout.getvalue()
+                print(stdout)
+
+            conn_id = 'CONN_ID3'
+            with create_session() as session:
+                current_conn = session.query(Connection).filter(Connection.conn_id == conn_id).first()
+                self.assertEqual(expected_connection_uris[conn_id][1],
```

Comment:
What do you think to compare results for get_uri method?

Reply:
Yup, make sense, updated in the PR.